cmd/main.go

```diff
@@ -74,6 +76,7 @@ func main() {
 	flag.StringVar(&gardenerProjectName, "gardener-project-name", "gardener-project", "Name of the Gardener project")
 	flag.Float64Var(&minimalRotationTimeRatio, "minimal-rotation-time", defaultMinimalRotationTimeRatio, "The ratio determines what is the minimal time that needs to pass to rotate certificate.")
 	flag.DurationVar(&expirationTime, "kubeconfig-expiration-time", defaultExpirationTime, "Dynamic kubeconfig expiration time")
+	flag.BoolVar(&enableProvisioning, "provisioning-enabled", defaultProvisioningEnabled, "Feature flag for all provisioning functionalities")
```

Comment:
Should be also documented in https://github.com/kyma-project/infrastructure-manager/tree/main/docs#configuration